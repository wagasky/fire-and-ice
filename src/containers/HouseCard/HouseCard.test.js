// import React from 'react';
// import { shallow } from 'enzyme';
// import { HouseCard, mapStateToProps, mapDispatchToProps } from './HouseCard';

// describe('HouseCard', () => {
//   let wrapper;

//   beforeEach(() => {
//     wrapper = shallow(<HouseCard />);
//   });

//   it('should match the snapshot', () => {
//     expect(wrapper).toMatchSnapshot();
//   });

//   // it('should map to the store correctly', () => {

//   // });

//   // it('should call the dispatch function when using a function to mapDispatchToProps', () => {

//   // });
// })