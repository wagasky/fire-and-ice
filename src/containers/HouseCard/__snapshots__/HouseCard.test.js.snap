// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HouseCard should match the snapshot 1`] = `undefined`;
